src/dbnode/encoding/scheme.go

```diff
@@ -44,12 +44,14 @@ var (
 
 	// TODO(xichen): set more reasonable defaults once we have more knowledge
 	// of the use cases for time units other than seconds.
-	defaultTimeEncodingSchemes = map[xtime.Unit]TimeEncodingScheme{
-		xtime.Second:      newTimeEncodingScheme(defaultNumValueBitsForBuckets, 32),
-		xtime.Millisecond: newTimeEncodingScheme(defaultNumValueBitsForBuckets, 32),
-		xtime.Microsecond: newTimeEncodingScheme(defaultNumValueBitsForBuckets, 64),
-		xtime.Nanosecond:  newTimeEncodingScheme(defaultNumValueBitsForBuckets, 64),
-	}
+	defaultTimeEncodingSchemes = NewTimeEncodingSchemes(
```

Comment:
I think it may be better to leave it as a `map[xtime.Unit]TimeEncodingScheme` and then in `encoding.newOptions` set `timeEncodingSchemes: NewTimeEncodingSchemes(defaultTimeEncodingSchemes),`

Reply:
Makes sense